scss/trachacks.scss

```diff
@@ -259,6 +279,10 @@ div[role="main"]{
     }
   }
 
+  #ctxtnav {
+    display: none;
```

Comment:
What's the effect of this change? From my (limited) testing, the `ctctnav` element is not present on the ticket list (I've seen it only on wiki pages)